packages/nodes-base/nodes/SyncroMSP/v1/actions/contact/add/description.ts

```diff
@@ -0,0 +1,109 @@
+import {
+	ContactProperties,
+} from '../../Interfaces';
+
+export const contactAddDescription: ContactProperties = [
+	{
+		displayName: 'Customer ID',
```

Comment:
Can we use `loadOptions` to fetch the Customer list so we avoid our users having to search the ID?